Python Examples/TestMonitor/CreateResultsAndSteps/create_results_and_steps.py

```diff
@@ -0,0 +1,246 @@
+    for voltage in range(0, 10):
+            # Simulate obtaining a power measurement.
+            power, inputs, outputs = measure_power(current, voltage)
+
+            # Test the power measurement.
+            if power < low_limit or power > high_limit:
+                status = {
+                    "statusType": "FAILED",
+                    "statusName": "Failed"
+                }
+            else:
+                status = {
+                    "statusType": "PASSED",
+                    "statusName": "Passed"
+                }
+            test_parameters = build_power_measurement_params(power, low_limit, high_limit, status)
+
+            # Generate a child step to represent the power output measurement.
+            measure_power_output_step_data = generate_step_data(
+                "Measure Power Output", "NumericLimit", inputs, outputs, test_parameters, status
+            )
+            # Create the step on the SystemLink enterprise.
+            measure_power_output_step_data["parentId"] = parent_step["stepId"]
+            measure_power_output_step_data["resultId"] = result_id
+            response = test_data_manager_client.create_steps(steps=[measure_power_output_step_data])
+            measure_power_output_step = response["steps"][0]
+
+            # If a test in the sweep fails, the entire sweep failed.  Mark the parent step accordingly.
+            if status["statusType"] == "FAILED":
+                # Update the parent test step's status on the SystemLink enterprise.
+                response = update_step_status(parent_step, "Failed")
+                parent_step = response["steps"][0]
+    
+    # Update the step status if the status is still running.
+    if parent_step["status"]["statusType"] == "RUNNING":
+        response = update_step_status(parent_step, "Passed")
+        parent_step = response["steps"][0]
+    return parent_step
+
+
+def get_test_result():
+    test_result = {
+        "programName": "Power Test",
+        "status": {
+            "statusType": "RUNNING",
+            "statusName": "Running"
+        },
+        "systemId": None,
+        "hostName": None,
+        "properties":None,
+        "serialNumber": str(uuid.uuid4()),
+        "operator": "John Smith",
+        "partNumber": "NI-ABC-123-PWR1",
+        "fileIds":None,
+        "startedAt": str(datetime.datetime.now()),
+        "totalTimeInSeconds": 0.0
+    }
+
+    return test_result
+
+
+def create_steps(test_result):
+    # Set test limits
+    low_limit = 0
+    high_limit = 70
+
+    """
+    Simulate a sweep across a range of electrical current and voltage.
+    For each value, calculate the electrical power (P=IV).
+    """
+    for current in range(0, 10):
+        voltage_sweep_step = create_parent_step(test_result["id"])
+        voltage_sweep_step = create_child_steps(voltage_sweep_step, test_result["id"], current, low_limit, high_limit)
```

Comment:
You're not using this second set of `voltage_sweep_step`

Reply:
removed the second voltage_sweep_step